content/news/2013/03/2013-03-14-nci-quitpal-app.md

```diff
@@ -7,7 +7,6 @@ topics:
   - mobile
   - mobile-apps
   - mobile-gov
-  - NCI
   - nih
```

Comment:
Is this an agency that we've decided to keep?

Reply:
@mejiaj  Is in part 2, https://github.com/GSA/digitalgov.gov/pull/7140